ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
```

Comment:
I suggest squashing this commit into cecf1e13ea940f7a75fa29613a5f2ef00637fe36 to reduce code churn